extensions/objects/templates/image-editor.component.ts

```diff
@@ -101,6 +101,8 @@ export class ImageEditorComponent implements OnInit, OnChanges {
   MODE_UPLOADED = 2;
   MODE_SAVED = 3;
 
+  IMAGE_UPLOADING = false;
```

Comment:
variable names should be lowercase. Use UPPERCASE only for constants. As this is a boolean, the name should be `imageIsUploading`.